src/parser/jobs/rdm/modules/index.tsx

```diff
@@ -7,12 +7,19 @@ import {DualCast} from './Dualcast'
 import {Embolden} from './Embolden'
 import {GeneralCDDowntime} from './GeneralCDDowntime'
 import {Interrupts} from './Interrupts'
+import {MagickedSwordplay} from './MagickedSwordplay'
+import {Manafication} from './Manafication'
 import {ManaGauge} from './ManaGauge'
 import {ManaStackGauge} from './ManaStackGauge'
 import {MeleeCombos} from './MeleeCombos'
+import {Prefulgence} from './Prefulgence'
 import {Procs} from './Procs'
 import {Swiftcast} from './Swiftcast'
+import {ViceOfThorns} from './ViceOfThorns'
 import {Weaving} from './Weaving'
+//For now commenting this out, it's uncertain if we want GrandImpact to be within Procs or on its own.
+//For now it'll be contained within Procs
+//import { GrandImpact } from './GrandImpact'
```

Comment:
See comments on Procs module, remove commented code and unused modules as necessary.

Reply:
I went ahead and removed it here, it'll be simple enough to re-add the 2 lines.